autoload/search_pulse.vim

```diff
@@ -20,7 +20,7 @@ func! s:ScrubPattern(s)
 endf
 
 func! s:Initialize()
-  let gui_running = has('gui_running')
+  let gui_running = has('gui_running') || (&termguicolors == 1)
```

Comment:
Please check `has('termguicolors')` before calling `&termguicolors` otherwise this will generate an error if vim wasn't compiled with +termguicolors. The example below shows the error:

```
:if &foo | echo 42 | endif
E113: Unknown option: foo
E15: Invalid expression: &foo | echo 42 | endif
```